### PR TITLE
Addition of missing params

### PR DESCRIPTION
**Pull request by pilasou**
Adding missing params and their french translations for : licence approval, SHERPA,  external sources and processes.

### CODE REVIEW

File: src/assets/i18n/fr.json5 [hunk 1]
```diff
@@ -4016,6 +4019,32 @@
...
   "process.overview.new": "Nouveau",
+  
+   // "process.overview.table.actions": "Actions",
+   "process.overview.table.actions": "Actions",
+  
+  // "process.overview.delete": "Delete {{count}} processes",
+  "process.overview.delete": "Supprimer {{count}} processus",
+
+  // "process.overview.delete.processing": "{{count}} process(es) are being deleted. Please wait for the deletion to fully complete. Note that this can take a while.",
+  "process.overview.delete.processing": "{{count}} processus sont en train d'être supprimer. Patientez jusqu'à ce que la suppression soit terminée. Notez que cela peut prendre un certain temps.",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:4030]: "process.overview.delete.processing": "{{count}} processus sont en train d'être **supprimés**. (...)"

File: src/assets/i18n/fr.json5 [hunk 2]
```diff
@@ -5676,6 +5732,95 @@
...
+  "submission.sections.sherpa.publication.information.romeoPub": "Romeo Pub",
+
+  // "submission.sections.sherpa.publication.information.zetoPub": "Zeto Pub",
+  "submission.sections.sherpa.publication.information.zetoPub": "Zeto Pub",
+
+  // "submission.sections.sherpa.publisher.policy": "Publisher Policy",
+  "submission.sections.sherpa.publisher.policy": "Politique de l'éditeur",
+
+  // "submission.sections.sherpa.publisher.policy.description": "The below information was found via Sherpa Romeo. Based on the policies of your publisher, it provides advice regarding whether an embargo may be necessary and/or which files you are allowed to upload. If you have questions, please contact your site administrator via the feedback form in the footer.",
+  "submission.sections.sherpa.publisher.policy.description": "L'information ci-dessous provient de Sherpa Romeo. Elle vous permet de savoir quelle version vous êtes autorisé-e à déposer et si une restriction de diffusion (embargo) doit être appliquée. Si vous avez des questions, contactez votre administrateur en utilisant le formulaire en pied de page.",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:5770]: (...) contactez votre administrateur-trice en utilisant le formulaire en pied de page.

File: src/assets/i18n/fr.json5 [hunk 3]
```diff
@@ -5676,6 +5732,95 @@
...
+  "submission.sections.sherpa.publisher.policy.embargo": "Embargo",
+
+  // "submission.sections.sherpa.publisher.policy.noembargo": "No Embargo",
+  "submission.sections.sherpa.publisher.policy.noembargo": "Aucun embargo",
+
+  // "submission.sections.sherpa.publisher.policy.nolocation": "None",
+  "submission.sections.sherpa.publisher.policy.nolocation": "Aucun",
+
+  // "submission.sections.sherpa.publisher.policy.license": "License",
+  "submission.sections.sherpa.publisher.policy.license": "License",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:5791]: Licence

File: src/assets/i18n/fr.json5 [hunk 4]
```diff
@@ -5759,6 +5904,12 @@
 
   // "submission.workflow.tasks.pool.show-detail": "Show detail",
   "submission.workflow.tasks.pool.show-detail": "Afficher le détail",
+  
+  // "submission.workspace.generic.view": "View",
+  "submission.workspace.generic.view": "Voir",
+
+  // "submission.workspace.generic.view-help": "Select this option to view the item's metadata.",
+  "submission.workspace.generic.view-help": "Sélecionner cette option pour voir les métadonnées de l'item.",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:5912]: Sélectionner

File: src/assets/i18n/fr.json5 [hunk 5]
```diff
@@ -5676,6 +5732,95 @@
...
+  "submission.sections.sherpa.publisher.policy.description": "L'information ci-dessous provient de Sherpa Romeo. Elle vous permet de savoir quelle version vous êtes autorisé-e à déposer et si une restriction de diffusion (embargo) doit être appliquée. Si vous avez des questions, contactez votre administrateur en utilisant le formulaire en pied de page.",
+
+  // "submission.sections.sherpa.publisher.policy.openaccess": "Open Access pathways permitted by this journal's policy are listed below by article version. Click on a pathway for a more detailed view",
+  "submission.sections.sherpa.publisher.policy.openaccess": "Les voies de libre accès autorisées par la politique de cette revue sont listées ci-dessous par version d'article. Cliquez sur l'une des voies pour plus de détails.",
+
+  // "submission.sections.sherpa.publisher.policy.more.information": "For more information, please see the following links:",
+  "submission.sections.sherpa.publisher.policy.more.information": "Pour plus d'information, cliquez sur le lien suivant:",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:5776]: add a space before colon => "(...) sur le lien suivant :"

File: src/assets/i18n/fr.json5 [hunk 6]
```diff
@@ -4016,6 +4019,32 @@
...
+  "process.overview.delete.body": "Êtes vous sûr de vouloir supprimer {{count}} processus?",
+
+  // "process.overview.delete.header": "Delete processes",
+  "process.overview.delete.header": "Supprimer les processus",
+
+  // "process.bulk.delete.error.head": "Error on deleteing process",
+  "process.bulk.delete.error.head": "Erreur lors de la suppression de processus",
+
+  // "process.bulk.delete.error.body": "The process with ID {{processId}} could not be deleted. The remaining processes will continue being deleted. ",
+  "process.bulk.delete.error.body": "Le processus numéro {{processId}} n'a pu être supprimé. Les processus restants continuent à être supprimés. ",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:4042]: For the second phrase, I would rather go for "Les processus restants continueront à être supprimés."

File: src/assets/i18n/fr.json5 [hunk 7]
```diff
@@ -4016,6 +4019,32 @@
...
+   "process.overview.table.actions": "Actions",
+  
+  // "process.overview.delete": "Delete {{count}} processes",
+  "process.overview.delete": "Supprimer {{count}} processus",
+
+  // "process.overview.delete.processing": "{{count}} process(es) are being deleted. Please wait for the deletion to fully complete. Note that this can take a while.",
+  "process.overview.delete.processing": "{{count}} processus sont en train d'être supprimer. Patientez jusqu'à ce que la suppression soit terminée. Notez que cela peut prendre un certain temps.",
+
+  // "process.overview.delete.body": "Are you sure you want to delete {{count}} process(es)?",
+  "process.overview.delete.body": "Êtes vous sûr de vouloir supprimer {{count}} processus?",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:4033]: "Êtes vous certain-e de vouloir (...)"   +  add space before question mark

**Review by mhvezina — CHANGES_REQUESTED** [review]
Thank you @pilasou for this translation. One comment though: I would use the infinitive and not the imperative tense to continue with the choices made with the existing translation (e.g. "Cliquer" instead of "Cliquez"). What do you think about this?

File: src/assets/i18n/fr.json5 (hunk 7 above)
Comment (pilasou) [reply src/assets/i18n/fr.json5]: Done, added to new commit

File: src/assets/i18n/fr.json5 (hunk 6 above)
Comment (pilasou) [reply src/assets/i18n/fr.json5]: Done, added to new commit

**Review by tdonohue — APPROVED** [review]
👍 Thanks @pilasou !  It looks like all feedback from @mhvezina was addressed.  I also gave this a quick test just to verify I see the new French translations, and I do.  Merging this

**Pull request merged**